### PR TITLE
Add tabby-codellama-7B

### PR DESCRIPTION
**Pull request by Janaka-Steph**
Missing "Hardware Requirements" and "Limitations and Biases" sections in readme.

### CODE REVIEW

File: coder-tabby-codellama-7b/manifest.json [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+    "beta": true,
+    "icon": "",
+    "modelInfo": {
+        "memoryRequirements": 3204
+    },
+    "interfaces": [
+        "coder"
+    ],
+    "volumePath": "/data",
+    "command": "serve --model TabbyML/CodeLlama-7B --device cuda",
```
Comment (filopedraz) [inline coder-tabby-codellama-7b/manifest.json:16]: Cuda?

File: coder-tabby-codellama-7b/manifest.json [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+    "documentation": "",
+    "beta": true,
+    "icon": "",
+    "modelInfo": {
+        "memoryRequirements": 3204
+    },
+    "interfaces": [
+        "coder"
+    ],
+    "volumePath": "/data",
```
Comment (filopedraz) [inline coder-tabby-codellama-7b/manifest.json:15]: Is volumePath necessary?

**Review by filopedraz — CHANGES_REQUESTED** [review]

File: coder-tabby-codellama-7b/manifest.json (hunk 1 above)
Comment (Janaka-Steph) [reply coder-tabby-codellama-7b/manifest.json:16]: `device` takes either `cuda`, `cpu` or `metal`.
It seems that `--device metal` is appropriate for Apple M1/M2 https://github.com/TabbyML/tabby/blob/cc83e4d2691f5438a236d4bb735ea6b9da436234/website/docs/extensions/troubleshooting.md?plain=1#L81
But I am not sure which one is best to use.

File: coder-tabby-codellama-7b/manifest.json (hunk 2 above)
Comment (Janaka-Steph) [reply coder-tabby-codellama-7b/manifest.json:15]: I guess not

File: coder-tabby-codellama-7b/manifest.json (hunk 1 above)
Comment (tiero) [reply coder-tabby-codellama-7b/manifest.json:15]: yes Metal only, as we are on Mac only. Cuda is server/linux

File: coder-tabby-codellama-7b/manifest.json (hunk 2 above)
Comment (filopedraz) [reply coder-tabby-codellama-7b/manifest.json]: @tiero how do we handle volumes for binaries? Same issue will appear for vector stores.

Comment (filopedraz) [reply coder-tabby-codellama-7b/manifest.json]: Makes sense

**Pull request merged**